apps/freelance/src/pages/ExampleRootPage/index.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+
+import { Link } from 'react-router-dom';
+import { ExampleButton } from '@freelance/components';
+import { Wrapper, Title, Text } from './styles';
+
+import { useGetPokemonByNameQuery } from '../../redux/services/pokemon';
+import { useTranslation } from 'react-i18next';
+
+export default function ExampleRootPage() {
+  const { t } = useTranslation();
+
+  const { data, error, isLoading } = useGetPokemonByNameQuery('bulbasaur');
+
+  return (
+    <Wrapper>
+      <Title>{t('description.root page')}</Title>
+      <h2>{t('description.Welcome to React')}</h2>
+      <Text>{t('description.reusable component')}</Text>
```

Comment:
no spaces, never never never in constants
bad - description.reusable component
good - description.reusable_component, description.reusableComponent

Reply:
updated